src/hipscat_import/run_import.py

```diff
@@ -1,54 +1,76 @@
 """Import a set of non-hipscat files using dask for parallelization"""
 
+
 import hipscat.io.write_metadata as io
 import numpy as np
 from dask.distributed import Client, as_completed
 from hipscat import pixel_math
 from tqdm import tqdm
 
 import hipscat_import.map_reduce as mr
+import hipscat_import.resume_files as rf
 from hipscat_import.arguments import ImportArguments
 
 
 def _map_pixels(args, client):
     """Generate a raw histogram of object counts in each healpix pixel"""
 
+    raw_histogram = rf.read_histogram(args.tmp_path, args.highest_healpix_order)
```

Comment:
I think using the non shortened import might make this more readable, or shortening to something like `resume` instead of `rf`

Reply:
Done.